MIGRATION.md

```diff
@@ -6,6 +6,7 @@ API changes in MoveIt! releases
 - Extended the return value of `MoveitCommander.MoveGroup.plan()` from `trajectory` to a tuple of `(success, trajectory, planning_time, error_code)` to better match the C++ MoveGroupInterface ([790](https://github.com/ros-planning/moveit/pull/790/))
 - `moveit_rviz.launch`, generated by MSA, provides an argument `rviz_config` to configure the rviz config to be used. The old boolean config argument was dropped. ([1397](https://github.com/ros-planning/moveit/pull/1397))
 - Moved the example package `moveit_controller_manager_example` into [moveit_tutorials](https://github.com/ros-planning/moveit_tutorials)
+- Requests to `get_planning_scene` service without explicitly setting "components" now return full scene
```

Comment:
```suggestion
- Requests to `get_planning_scene` service without explicitly setting "components" now returns full scene
```

Reply:
No, I think this is correct, as requests is plural.

Reply:
You are right.